src/build-scripts/build_materialX_osl.py

```diff
@@ -14,15 +14,19 @@
 import os
 import sys
 import re
+import argparse
 
+# global defaults
+ARCH = 'linux64'
```

Comment:
I'm thinking that this python script should take the architecture (or, even better, just the paths to oslc and the shaders/MaterialX directory), which CMakeLists.txt can easily pass to it.

If the oslc path is not supplied, it should just assume 'oslc' is in the execution path, and if the build/shaders area isn't supplied, it should just assume the current dir.